packages/flutter_android_lifecycle/example/test/widget_test.dart

```diff
@@ -0,0 +1,27 @@
+// This is a basic Flutter widget test.
+//
+// To perform an interaction with a widget in your test, use the WidgetTester
+// utility that Flutter provides. For example, you can send tap and scroll
+// gestures. You can also use WidgetTester to find child widgets in the widget
+// tree, read text, and verify that the values of widget properties are correct.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:flutter_android_lifecycle_example/main.dart';
+
+void main() {
+  testWidgets('Verify Platform version', (WidgetTester tester) async {
+    // Build our app and trigger a frame.
+    await tester.pumpWidget(MyApp());
```

Comment:
Actually if we leave just this line (and make the Android code use `getLifecycle`) we should have our e2e test.

Reply:
This test is pretty close to being the e2e test you want but it's actually just a unit test that ships with the flutter create plugin template. It will need to call `ensureInitialized` to be runnable with `FlutterRunner` or `flutter drive`. We could add a TODO to that effect if we want to get it landed.

Reply:
widget_test.dart will require main.dart, which currently requires flutter_android_lifecycle.dart. Do you know which subset of these files we want to keep?